src/pybind/mgr/dashboard/model/feedback.py

```diff
@@ -12,7 +12,7 @@ class Feedback:
     class Project(Enum):
         dashboard = 46
         block = 9 #rbd
-        object = 10 # rgw
+        objects = 10 # rgw
```

Comment:
Why plural here?

Reply:
changed back to `object`. I did it because `object` looked like a keyword in my editor and I wanted to avoid any trouble because of it. But I check and it isn't a keyword in python.